examples/tree_2d_dgsem/elixir_ape_euler_co-rotating_vortex_pair.jl

```diff
@@ -0,0 +1,337 @@
+
+boundary_condition_euler = VortexPairSetup.BoundaryCondition(rho, (rho * c0^2 / gamma) / (gamma-1))
+
+semi_euler = SemidiscretizationHyperbolic(mesh, equations_euler, initial_condition_euler, solver,
+                                          boundary_conditions=boundary_condition_euler,
+                                          source_terms=sponge_layer_euler)
+
+###############################################################################
+# semidiscretization acoustic perturbation equations
+equations_ape = AcousticPerturbationEquations2D(v_mean_global=(13.0, 26.0), c_mean_global=39.0,
+                                                rho_mean_global=52.0) # global mean values will be overwritten
+
+sponge_layer_ape = VortexPairSetup.SpongeLayer(sponge_layer_min=(-135*r0, 100*r0, -135*r0, 100*r0),
+                                               sponge_layer_max=(-100*r0, 135*r0, -100*r0, 135*r0),
+                                               reference_values=(0.0,))
+
+semi_ape = SemidiscretizationHyperbolic(mesh, equations_ape, initial_condition_constant, solver,
+                                        boundary_conditions=boundary_condition_zero,
+                                        source_terms=sponge_layer_ape)
+
+###############################################################################
+# ODE solvers, callbacks etc. for averaging the flow field
+
+T_r = 8 * pi^2 * r0^2 / circulation # Rotational period of the vortex pair
+T_a = T_r / 2 # Acoustic period of the vortex pair
+
+# Create ODE problem
+tspan1 = (0.0, 400.0)
+ode_averaging = semidiscretize(semi_euler, tspan1)
+
+# At the beginning of the main loop, the SummaryCallback prints a summary of the simulation setup
+# and resets the timers
+summary_callback = SummaryCallback()
+
+analysis_interval=5000
+alive = AliveCallback(analysis_interval=analysis_interval)
+
+tspan_averaging = (50.0, 400.0)
+averaging_callback = AveragingCallback(semi_euler, tspan=tspan_averaging)
+
+cfl = 1.0
+stepsize_callback = StepsizeCallback(cfl=cfl)
+
+callbacks_averaging = CallbackSet(summary_callback, alive, averaging_callback, stepsize_callback)
+
+###############################################################################
+# run simulation for averaging the flow field
+
+# OrdinaryDiffEq's `solve` method evolves the solution in time and executes the passed callbacks
+sol_averaging = solve(ode_averaging, CarpenterKennedy2N54(williamson_condition=false),
+                      dt=1.0, # solve needs some value here but it will be overwritten by the stepsize_callback
+                      save_everystep=false, callback=callbacks_averaging);
+
+# Print the timer summary
+summary_callback()
+
+
+###############################################################################
+# set up coupled semidiscretization
+
+source_region(x) = sum(x.^2) < 6.0^2 ? true : false # calculate sources within radius 6 around origin
```

Comment:
Can this be simplified to something like this?

```suggestion
source_region(x) = sum(x.^2) < 6.0^2 # calculate sources within radius 6 around origin
```